diff --git a/lib/metrics.js b/lib/metrics.js
index 9e41d16678d11e3b77e1af881774fac4561828cd..ae5cea5ef957815e8d4a80442fb991e4438590d1 100644
--- a/lib/metrics.js
+++ b/lib/metrics.js
@@ -7,7 +7,8 @@ const {
   collectDefaultMetrics,
 } = require('prom-client');
 
-const { cpu, mem, netstat } = osu = require('node-os-utils');
+const osu = require('node-os-utils');
+const { cpu, mem, netstat } = osu;
 
 module.exports = function configure(config) {
   // eslint-disable-next-line max-len
